test/basicConfiguration.spec.js

```diff
@@ -94,4 +94,35 @@ describe('Basic configuration', function() {
 
   });
 
+  it('should not have test mode ribbon by default', function(done) {
+
+    var appName = 'my-test-app';
+
+    createWidget();
+
+    expect($('.corner-ribbon')).not.toBeInDOM();
+    expect($('.corner-ribbon')).not.toBeVisible();
+    done();
+
+  });
+
+  it('should have test mode ribbon when set', function(done) {
+
+    var appName = 'my-test-app';
+    
+    var config = {
+      showTestRibbon: true
+    }
+    createWidget(config);
+
+    setTimeout(function() {
+
+      expect($('.corner-ribbon')).toBeInDOM();
+      expect($('.corner-ribbon')).toBeVisible();
+      done();
+
+    }, 200);
+
+  });
+
```

Comment:
Perhaps make yet another test where it renders the ribbon after a FindTime call has been made, just to make sure that this works too. Tip: you can define a mock response in `utils/mockAjax.js` and use it

Reply:
rgr